spec/sidekiq/lighthouse/poll_form526_pdf_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Lighthouse::PollForm526Pdf, type: :job do
+  subject { described_class }
+
+  before do
+    Sidekiq::Job.clear_all
+    allow_any_instance_of(Auth::ClientCredentials::Service).to receive(:get_token).and_return('fake_access_token')
+  end
+
+  let(:user) { FactoryBot.create(:user, :loa3) }
```

Comment:
Nice clean up. These aren't needed any longer either.

Reply:
Killed 'em! Thanks for your help on this, there were a lot more weeds to ~~poll~~ pull than I realized!